tests/integration.rs

```diff
@@ -4895,22 +4895,24 @@ mod tests {
         }
 
         #[cfg_attr(not(feature = "mmio"), test)]
-        #[cfg(target_arch = "x86_64")]
         fn test_counters() {
             let mut focal = UbuntuDiskConfig::new(FOCAL_IMAGE_NAME.to_string());
             let guest = Guest::new(&mut focal);
             let api_socket = temp_api_path(&guest.tmp_dir);
 
-            let mut child = GuestCommand::new(&guest)
-                .args(&["--cpus", "boot=1"])
+            let mut cmd = GuestCommand::new(&guest);
+            cmd.args(&["--cpus", "boot=1"])
                 .args(&["--memory", "size=512M"])
                 .args(&["--kernel", guest.fw_path.as_str()])
                 .default_disks()
                 .args(&["--net", guest.default_net_string().as_str()])
                 .args(&["--api-socket", &api_socket])
-                .capture_output()
-                .spawn()
-                .unwrap();
+                .capture_output();
+
+            #[cfg(target_arch = "aarch64")]
+            cmd.args(&["--cmdline", DIRECT_KERNEL_BOOT_CMDLINE]);
```

Comment:
@michael2012z Since you require direct kernel boot, don't you have to change `--kernel` option as well for `aarch64`?

Reply:
Indeed `aarch64` can only boot from direct kernel. So I set `fw_path` direct kernel in some previous commit for simplicity. https://github.com/cloud-hypervisor/cloud-hypervisor/blob/master/tests/integration.rs#L588

Reply:
Oh I see, that's a bit confusing, but I understand how this simplifies things for every test using the FW.